### PR TITLE
support.steampowered.com is dark

### PR DESCRIPTION
**Pull request by nathanielcwm**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -727,6 +727,7 @@ sumo.app
 sunxdcc.com
 supabase.io
 supinic.com
+support.steampowered.com
```
Comment (Gusted) [inline src/config/dark-sites.config:730]: Support? I'm getting a redirect to help.steampowered.com

Comment (nathanielcwm) [reply src/config/dark-sites.config:730]: idk the actual help pages seem to still be on the support subdomain eg:
https://support.steampowered.com/kb_article.php?ref=5414-tfbn-1352
https://support.steampowered.com/kb_article.php?ref=7480-WUSF-3601
https://support.steampowered.com/kb_article.php?ref=4044-QDHJ-5691

Comment (nathanielcwm) [reply src/config/dark-sites.config:730]: well help.steampowered.com is dark too lol

**Review by Gusted — APPROVED** [review]
Looking good to me!

**Pull request merged**